src/menu.scss

```diff
@@ -5,351 +5,306 @@
 $block: #{$fd-namespace}-menu;
 
 .#{$block} {
-  // LOCAL VARS (set all themeable properties, always include !default)
-  // items
-  $fd-menu-item-color: var(--sapList_TextColor) !default;
-  $fd-menu-item-color--active: var(--sapList_Active_TextColor) !default;
-
-  $fd-menu-item-background-color: var(--sapList_Background) !default;
-  $fd-menu-item-background-color--hover: var(--sapList_Hover_Background) !default;
-  $fd-menu-item-background-color--active: var(--sapList_Active_Background) !default;
-  $fd-menu-item-background-color--selected: var(--sapList_SelectionBackgroundColor) !default;
-  $fd-menu-item-background-color--selected-hover: var(--sapList_Hover_SelectionBackground) !default;
-
-  $fd-menu-separator-color: var(--sapGroup_ContentBorderColor) !default;
+  // Menu
+  $fd-menu-border-radius: var(--sapElement_BorderCornerRadius) !default;
+
+  // Menu Item
+  $fd-menu-item-color-active: var(--sapList_Active_TextColor) !default;
+
+  // Menu Link
+  $fd-menu-icon-width: 2.25rem !default; // to account for item right padding of 16px or 1rem so 2.25rem-1rem = 1.25rem
+  $fd-menu-icon-width-compact: 2rem !default;
+  $fd-menu-icon-width-mobile: 2.75rem !default;
+  $fd-menu-icon-after-margin: 0 -1rem 0 0 !default;
+  $fd-menu-icon-before-margin: 0 0 0 -0.75rem !default;
+  $fd-menu-icon-before-margin-compact: 0 0 0 -0.5rem !default;
+  $fd-menu-icon-after-margin-compact: 0 -0.75rem 0 0 !default;
+  $fd-menu-icon-before-margin-mobile: 0 0 0 -1rem !default;
+  $fd-menu-icon-after-margin-mobile: 0 -1rem 0 0 !default;
```

Comment:
I don't like those kind of variables. I would go more for `$fd-menu-icon-margin(-mobile)` with `-x`, or `-y`.
so ex. `$fd-menu-icon-margin-x: 1rem`

Reply:
Let's have a talk about it

Reply:
I guess that one is resolved according to our conversation.